source/math/FloatBox2D.ooc

```diff
@@ -87,18 +87,10 @@ FloatBox2D: cover {
 		This new(left, top, width, height)
 	}
 	union: func ~box (other: This) -> This { // Rock bug: Union without suffix cannot be used because the C name conflicts with keyword "union"
-		left := Float minimum(this left, other left)
-		top := Float minimum(this top, other top)
-		width := Float maximum(0, (Float maximum(this right, other right) - left))
-		height := Float maximum(0, (Float maximum(this bottom, other bottom) - top))
-		This new(left, top, width, height)
+		This new(this leftTop minimum(other leftTop), this rightBottom maximum(other rightBottom))
```

Comment:
Is a `Size` cover guaranteed to be nonnegative? The previous version made sure width and height were >= 0, that doesn't happen here.